### PR TITLE
docs: update Akamai platform docs

### PR DESCRIPTION
**Pull request by eljohnson92**
# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md [hunk 1]
```diff
@@ -55,21 +57,27 @@ Run the following commands to create three control plane nodes:
...
 for id in $(seq 3); do
   linode_label="talos-control-plane-${id}"
+
   # create linode
+
   linode-cli linodes create  \
     --no-defaults \
-    --root_pass securepass123! \
-    --type g6-standard-4 \
+    --root_pass ${ROOT_PW} \
```
Comment (smira) [inline website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md:72]: Talos doesn't have a concept of the root password, so I guess this gets ignored/dropped?

Comment (smira) [reply website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md:72]: if so, we might need a note about that

Comment (eljohnson92) [reply website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md:72]: Yes, it is a linode API requirement. I can add a note that this is required but not used

Comment (eljohnson92) [reply website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md:72]: I've updated the docs with a note calling this out

**Review by smira — APPROVED** [review]

**Pull request merged**